plonky2/src/hash/poseidon.rs

```diff
@@ -320,6 +371,28 @@ pub trait Poseidon: PrimeField64 {
         }
     }
 
+    #[inline(always)]
+    #[unroll_for_loops]
+    fn partial_first_constant_layer_packed<
```

Comment:
Very very nit, for naming consistency:
```suggestion
    fn partial_first_constant_layer_packed_field<
```